File: ClassProject/Ex084GSONlibrary/app/src/main/java/com/inah_wook/ex084gsonlibrary/Person.java
package com.inah_wook.ex084gsonlibrary;

public class Person {
    // 기능없는 데이터용 클래스
    // json의 name과 age 만 저장하는 데이터 분산을 막으려고 클래스 하나 만든 거.

    String name;
    int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public Person() {
    }
}

File: ClassProject/Ex095KotlinHello/app/src/main/java/com/inah_wook/ex095kotlinhello/Person.kt
package com.inah_wook.ex095kotlinhello

open class Person constructor(val name:String , val age:Int){  //constructor() :  주생성자 -> val : 맴버변수로 아예 생성
    init {
        println(" create Person obj")
    }

    open fun show(){
        println(" name : $name , age : $age ")
    }




}